inst/run_orderly_demo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Generally I think it's nicer to avoid using extensions on scripts as then we can reimplement them in another language later without having to update uses. Your editor should pick up syntax highlighting properly based on the first line